OJS.Workers.ExecutionStrategies/BaseCodeExecutionStrategy.cs

```diff
@@ -61,6 +61,7 @@ protected void SaveZipSubmission(byte[] submissionContent, string directory)
                 TimeUsed = (int)processExecutionResult.TimeWorked.TotalMilliseconds,
                 MemoryUsed = (int)processExecutionResult.MemoryUsed,
                 IsTrialTest = test.IsTrialTest,
+                CheckerDetails = new CheckerDetails(),
```

Comment:
Can you initialize it in the model directly with `{ get; set; } = new();`

Reply:
FIxed.